src/commands/sbom/language.ts

```diff
@@ -32,7 +38,7 @@ export const getLanguageFromComponent = (component: any): DependencyLanguage | u
     }
   }
 
-  console.debug(`language for component ${componentName} not found`)
+  console.debug(`language for component ${componentName} (purl: ${purlDisplay}) not found or supported`)
```

Comment:
[readability] imo this reads better

```suggestion
  console.debug(`language not detected for ${componentName} with due to purl (${purlDisplay})`)
```